src/pint/pintk/plk.py

```diff
@@ -243,7 +243,10 @@ def clear_grid(self):
             widget.grid_forget()
 
     def changedRMCheckBox(self):
-        log.info("Random Models set to %d" % (self.var.get()))
+        if self.var.get() == 1:
+            log.info("Random Models turned on.")
```

Comment:
Should this be info or debug? It seems like the UI should be making it pretty clear.